x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.ts

```diff
@@ -221,7 +221,12 @@ export const RelevanceTuningLogic = kea<
     ],
   }),
   selectors: ({ selectors }) => ({
-    schemaFields: [() => [selectors.schema], (schema: Schema) => Object.keys(schema)],
+    schemaFields: [
+      () => [selectors.schema],
+      (schema: AdvancedSchema) => Object.entries(schema).reduce((fields: string[], [fieldName, {capabilities: fieldCapabilies}]) => {
+        return (fieldCapabilies.fulltext || fieldCapabilies.boost) ? [ ...fields, fieldName ] : fields;
```

Comment:
The backend now indicates if a field can be used in full-text search (`capabilities.fulltext`) or as a boost (`capabilities.boost`).

Fields that can not be boosted or searched are filtered out here.

By default, the backend [code](https://github.com/elastic/ent-search/pull/6679) return `false` for these capabilities of nested fields and subfields.